### DISCUSSION TITLE
BOM Import

### DISCUSSION THREAD

**Discussion by bgrouns** [blueseerERP/blueseer#89] [Q&A] [answered]
Is it possible import BOM structures, or is there a way to associate multiple purchased items with a parent manufactured item all at one time instead of one at a time?

**Comment by vaughnte** [comment, marked as answer]
Yes...you can mass load a data import file using the Mass Data Import menu.    The required fields for the file to load can be determined by choosing the 'BOM Master' option and clicking 'define'.   There is also a sample file in c:\blueseer\data\sampledir.    Also, check the documentation for Mass Data Import.   If you have any issues, send an email to the support email located on the contact page at blueseer.com.   There is a documentation link on the site as well.

**Comment by bgrouns** [comment]
Got it.  Thank you, sir!
